ui/imports/shared/popups/send/views/qmldir

```diff
@@ -7,6 +7,5 @@ NetworkSelector 1.0 NetworkSelector.qml
 NetworksSimpleRoutingView 1.0 NetworksSimpleRoutingView.qml
 RecipientView 1.0 RecipientView.qml
 SendModalFooter 1.0 SendModalFooter.qml
-TabAddressSelectorView 1.0 TabAddressSelectorView.qml
```

Comment:
Shouldn't you add the new component here (since you renamed it)?

Reply:
It's been added in `../panels/qmldir` since it's been renamed and moved!